etl/assets.py

```diff
@@ -83,43 +78,37 @@ def documents_of_wikipedia_articles_with_summaries(
 @asset
 def wikipedia_articles_embedding_store(
     documents_of_wikipedia_articles_with_summaries: DocumentTuple,
-    openai_settings: OpenaiSettings,
-    input_config: InputConfig,
+    pipeline_config: PipelineConfig,
     output_config: OutputConfig,
 ) -> None:
     """Materialize an asset of Wikipedia articles embeddings."""
 
-    parsed_input_config = input_config.parse()
-
     OpenaiEmbeddingPipeline(
-        openai_settings=openai_settings,
+        openai_settings=pipeline_config.openai_settings,
         output_config=output_config,
     ).create_embedding_store(
         documents=documents_of_wikipedia_articles_with_summaries.documents,
-        distance_strategy=parsed_input_config.distance_strategy,
```

Comment:
Discussed that these aren't needed when creating embeddings, only when calculating similarity scores.

Reply:
Yes, I have removed these parameters from `create_embedding_store`, and moved them to another config.